tests/evm/test_create.py

```diff
@@ -0,0 +1,374 @@
+        .call_context_write(CURRENT_CALL_ID, CallContextFieldTag.MemorySize, next_memory_size) \
+        .call_context_write(CURRENT_CALL_ID, CallContextFieldTag.ReversibleWriteCounter, caller_ctx.reversible_write_counter + 1) 
+    
+    # callee's call context
+    rw_dictionary \
+        .call_context_read(next_call_id, CallContextFieldTag.CallerId, CURRENT_CALL_ID) \
+        .call_context_read(next_call_id, CallContextFieldTag.TxId, 1) \
+        .call_context_read(next_call_id, CallContextFieldTag.Depth, 2) \
+        .call_context_read(next_call_id, CallContextFieldTag.CallerAddress, caller.address) \
+        .call_context_read(next_call_id, CallContextFieldTag.CalleeAddress, contract_address) \
+        .call_context_read(next_call_id, CallContextFieldTag.IsSuccess, is_success) \
+        .call_context_read(next_call_id, CallContextFieldTag.IsStatic, is_static) \
+        .call_context_read(next_call_id, CallContextFieldTag.IsRoot, False) \
+        .call_context_read(next_call_id, CallContextFieldTag.IsCreate, True) \
+        .call_context_read(next_call_id, CallContextFieldTag.CodeHash, RLC(EMPTY_CODE_HASH))
+    # fmt: on
+
+    tables = Tables(
+        block_table=set(Block().table_assignments(randomness)),
+        tx_table=set(),
+        bytecode_table=set(
+            chain(
+                caller_bytecode.table_assignments(randomness),
+                init_codes.table_assignments(randomness),
+            )
+        ),
+        rw_table=set(rw_dictionary.rws),
+        copy_circuit=copy_circuit.rows,
+    )
+
+    # verify_copy_table(copy_circuit, tables, randomness)
+
+    verify_steps(
+        randomness=randomness,
+        tables=tables,
+        steps=[
+            StepState(
+                execution_state=ExecutionState.CREATE2
+                if is_create2 == 1
+                else ExecutionState.CREATE,
+                rw_counter=rw_counter,
+                call_id=CURRENT_CALL_ID,
+                is_root=False,
+                is_create=True,
+                code_hash=caller_bytecode_hash.expr(),
+                program_counter=next_program_counter,
+                stack_pointer=stack_pointer,
+                gas_left=caller_ctx.gas_left,
+                memory_word_size=caller_ctx.memory_word_size,
+                reversible_write_counter=caller_ctx.reversible_write_counter,
+                aux_data=init_codes_hash,
+            ),
+            (
+                StepState(
+                    execution_state=ExecutionState.STOP,
+                    rw_counter=rw_dictionary.rw_counter,
+                    call_id=next_call_id,
+                    is_root=False,
+                    is_create=True,
+                    code_hash=caller_bytecode_hash.expr(),
+                    program_counter=0,
+                    stack_pointer=1024,
+                    gas_left=callee_gas_left,
+                    reversible_write_counter=2,
+                )
+            ),
```

Comment:
hmm I think next step should use `init_code_hash`, so I think we don't really need `aux_data` above.
```suggestion
            (
                StepState(
                    execution_state=ExecutionState.PUSH,
                    rw_counter=rw_dictionary.rw_counter,
                    call_id=next_call_id,
                    is_root=False,
                    is_create=True,
                    code_hash=init_codes_hash.expr(),
                    program_counter=0,
                    stack_pointer=1024,
                    gas_left=callee_gas_left,
                    reversible_write_counter=2,
                )
            ),
```

Reply:
fixed in [a6ed17f](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/355/commits/a6ed17f2817ad1a263c7dce24659305e3c996ffa) and also merged word hi/lo.